### DISCUSSION TITLE
Why responseSingle? This loads the entire response on memory. How can we stream the response without aggregating the response on memory?

### DISCUSSION THREAD

**Discussion by sunny-shaw** [resteasy/resteasy#3257] [General]
https://github.com/resteasy/resteasy/blob/4e41a4386f00b5291a5336397ff64716ebc07251/resteasy-client-reactor-netty/src/main/java/org/jboss/resteasy/client/jaxrs/engines/ReactorNettyClientHttpEngine.java#L123

**Comment by jamezp** [comment]
This does seem odd and I think an issue should be filed at https://issues.redhat.com/browse/RESTEASY for this.
